.code-samples.meilisearch.yaml

```diff
@@ -660,22 +660,24 @@ geosearch_guide_sort_usage_2: |-
       "rating:desc",
     },
   })
-document_guide_create_index_primary_key: |-
+primary_field_guide_create_index_primary_key: |-
```

Comment:
I cannot make a suggestion but it should be `books` not `movies`

https://github.com/meilisearch/documentation/blob/9d414bfe3d2984b6280898964f1594abcc1c1fc8/.code-samples.meilisearch.yaml#L764